apps/token/src/components/vega-wallet/vega-wallet.tsx

```diff
@@ -37,7 +37,7 @@ export const VegaWallet = () => {
   );
 
   return (
-    <section className="vega-wallet">
+    <section className="vega-wallet" data-testid="vega-wallet">
```

Comment:
Do we need that, if there's already class named like that?

Reply:
Not really needed as such but the cypress best practice - favours adding data-* attributes.